<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clock_Main_Window</class>
 <widget class="QMainWindow" name="Clock_Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>402</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>402</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>402</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smail Clock</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_stopwatch">
       <attribute name="title">
        <string>秒表</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_stopwatch_outside">
        <property name="topMargin">
         <number>50</number>
        </property>
        <item row="0" column="0">
         <widget class="Clock_Number_Display_Form" name="clock_number_display_form_stopwatch" native="true">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>130</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>130</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_stopwatch_inside">
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_stopwatch_reset">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>重置</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_stopwatch_start_stop">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>开始</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_timer">
       <attribute name="title">
        <string>计时器</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_timer_outside">
        <property name="topMargin">
         <number>50</number>
        </property>
        <item row="0" column="0">
         <widget class="Clock_Number_Display_Form" name="clock_number_display_form_timer" native="true">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>130</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>130</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_timer_inside">
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_timer_start_stop">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>开始</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_timer_reset">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>重置</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_timer_set">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>设置时间</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_alarm_clock">
       <attribute name="title">
        <string>闹钟</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget_alarm_clock">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <attribute name="headerMinimumSectionSize">
           <number>100</number>
          </attribute>
          <column>
           <property name="text">
            <string>名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>次数</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>消息</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>命令</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>开关</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action_set"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_set">
   <property name="text">
    <string>首选项</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Clock_Number_Display_Form</class>
   <extends>QWidget</extends>
   <header>clock_number_display_form.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
